Gems/MotionMatching/Code/Source/CsvSerializers.cpp

```diff
@@ -63,8 +63,11 @@ namespace EMotionFX::MotionMatching
 
     ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-    bool PoseWriterCsv::Begin(const char* filename, ActorInstance* actorInstance)
+    bool PoseWriterCsv::Begin(const char* filename, ActorInstance* actorInstance, bool writePositions, bool writeRotations)
```

Comment:
suggestion: It might be nice rather than bools (that are harder to discern at the callsite) to use an enum mask so you can pass in something like... `ExportSetting::WriteRotations | ExportSettings::WritePositions` so it's much clearer to see what's happening (I think @kberg-amzn added a macro for doing typesafe enum masks too IIRC so you can avoid an unscoped enum if possible)